src/storages/ConfigStorage.sol

```diff
@@ -99,6 +100,8 @@ contract ConfigStorage is IConfigStorage, OwnableUpgradeable {
   mapping(uint256 marketIndex => uint256 minProfitDuration) public minProfitDurations;
   // If enabled, this market will used Adaptive Fee based on CEX orderbook liquidity depth
   mapping(uint256 marketIndex => bool isEnabled) public isAdaptiveFeeEnabledByMarketIndex;
+  // Min profit duration in steps based on trade size
+  StepMinProfitDuration[] public stepMinProfitDurations;
```

Comment:
step min duration is apply to all markets?

Reply:
yes